Name: Always Show Previous and Next Buttons
Version: 3.0.4-1
Author: Jason Robitaille
Description: The previous and next email buttons will be displayed always  (MIT license)

--- .orig/usr/palm/applications/com.palm.app.email/mail/source/MailApp.js
+++ /usr/palm/applications/com.palm.app.email/mail/source/MailApp.js
@@ -174,7 +174,7 @@
 				var accountId = currentFolder.accountId;
 				enyo.application.dashboardManager.setFilter(accountId, currentFolder._id);
 			}
-			this.$.body.hideNextPrev();
+			this.$.body.exposeNextPrev();
 		}
 	},
 	rendered: function() {
--- .orig/usr/palm/applications/com.palm.app.email/mail/source/Body.js
+++ /usr/palm/applications/com.palm.app.email/mail/source/Body.js
@@ -191,8 +191,8 @@
 			// {name: "moveToFolderCommandItem", icon: "../images/icons/toolbar-icon-moveto.png", onclick: "moveToFolderClick"},
 			// {kind: "Control", className:"toolbar-spacer"},
 			{name: "deleteCommandItem", icon: "../images/icons/toolbar-icon-delete.png",  onclick: "deleteClick"},
-			{kind: "Control", name: "navgroupSpacer", className:"toolbar-spacer", showing:false},
-			{kind:enyo.VFlexBox, name:"navgroup", style:"position:absolute;right:14px;margin:6px 0;", showing:false, width:"118px", components: [ 
+			{kind: "Control", name: "navgroupSpacer", className:"toolbar-spacer", showing:true},
+			{kind: "Control", className:"toolbar-spacer", width:"100px"},{kind:enyo.VFlexBox, name:"navgroup", style:"position:absolute;right:14px;margin:6px 0;", showing:true, width:"118px", components: [ 
 				{kind: "RadioGroup", name: "radioSelection", components: [
 					{icon: "../images/previous-email-icon.png", disabled:true, name:"prevButton", onclick: "goToPrevEmail"}, 
 					{icon: "../images/next-email-icon.png", disabled:true, name: "nextButton", onclick: "goToNextEmail"} 
